demos/object_detection_demo/python/README.md

```diff
@@ -276,6 +276,72 @@ To avoid disk space overrun in case of continuous input stream, like camera, you
 
 >**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, you can download OpenCV FFMPEG back end using the PowerShell script provided with the OpenVINO &trade; install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. The script should be run with administrative privileges if OpenVINO &trade; is installed in a system protected folder (this is a typical case). Alternatively, you can save results as images.
 
+## Running with OpenVINO Model Server
+
+You can also run this demo with model served in [OpenVINO Model Server](https://github.com/openvinotoolkit/model_server).
+
+Model Server is distributed as docker image, so you can pull it from Docker Hub with:
+
+```
+docker pull openvino/model_server
+```
+
+Use instruction from "Preparing to Run" section to download and convert the model and then place model files in the following directory structure:
+
+```
+<model_directory>
+└── <version_directory>
+    ├── model.bin
+    └── model.xml
+```
+
+For example, for face detection model, it could look like this:
+
+```
+face_detection
+└── 1
+    ├── face-detection-retail-0004.bin
+    └── face-detection-retail-0004.xml
+```
+
+Now you can run OpenVINO Model Server:
+```
+docker run -d -u $(id -u):$(id -g) -v <model_directory>:/model -p 9000:9000 openvino/model_server:latest \
+--model_path /model --model_name object_detection --port 9000
```

Comment:
```suggestion
--model_path /model --model_name face_detection --port 9000
```
It's better to continue the all following examples with the `face_detection` model name, for which we definied how the model directory structure looks like